crates/router/tests/connectors/multisafepay.rs

```diff
@@ -0,0 +1,438 @@
+use masking::Secret;
+use router::types::{self, api, storage::enums};
+
+use crate::{
+    connector_auth,
+    utils::{self, ConnectorActions},
+};
+
+#[derive(Clone, Copy)]
+struct MultisafepayTest;
+impl ConnectorActions for MultisafepayTest {}
+impl utils::Connector for MultisafepayTest {
+    fn get_data(&self) -> types::api::ConnectorData {
+        use router::connector::Multisafepay;
+        types::api::ConnectorData {
+            connector: Box::new(&Multisafepay),
+            connector_name: types::Connector::Multisafepay,
+            get_token: types::api::GetToken::Connector,
+        }
+    }
+
+    fn get_auth_token(&self) -> types::ConnectorAuthType {
+        types::ConnectorAuthType::from(
+            connector_auth::ConnectorAuthentication::new()
+                .multisafepay
+                .expect("Missing connector authentication configuration"),
+        )
+    }
+
+    fn get_name(&self) -> String {
+        "multisafepay".to_string()
+    }
+}
+
+static CONNECTOR: MultisafepayTest = MultisafepayTest {};
+
+// Cards Positive Tests
+// Creates a payment using the manual capture flow (Non 3DS).
+#[actix_web::test]
+async fn should_only_authorize_payment() {
```

Comment:
please check if the tests are passing and update the PR description with test status

Reply:
The connector does not supports Non 3DS flows and, capture and mandate are supported upon request to connector, so most unit tests will fail. But tested via. Postman. Adding link of [screenshots](https://drive.google.com/file/d/1M-Vu2XUsg8gsOgW25Du-vNTom4XDDun5/view?usp=sharing).